doc/whats-new.rst

```diff
@@ -181,6 +181,9 @@ Documentation
 
 Enhancements
 ~~~~~~~~~~~~
+- added backend for many Atmospheric data formats ranging including GEOS-Chem,
+  CAMx, NOAA arlpacked bit and many others.
```

Comment:
please add your name, if you're OK being credited for this PR.

Also, this section needs to be moved up on to docs for 0.10.4.

Reply:
Done.